ui/src/components/identities/IdentityForm.tsx

```diff
@@ -0,0 +1,254 @@
+import { Button, Card, Col, Divider, Form, Input, Row, Select, Typography, message } from "antd";
+import { useCallback, useEffect, useState } from "react";
+import { getSupportedNetwork } from "src/adapters/api/identities";
+import { IdentityFormData, identityFormDataParser } from "src/adapters/parsers/view";
+import { ErrorResult } from "src/components/shared/ErrorResult";
+import { LoadingResult } from "src/components/shared/LoadingResult";
+import { useEnvContext } from "src/contexts/Env";
+import { AppError, CredentialStatusType, IdentityType, Method, SupportedNetwork } from "src/domain";
+import {
+  AsyncTask,
+  hasAsyncTaskFailed,
+  isAsyncTaskDataAvailable,
+  isAsyncTaskStarting,
+} from "src/utils/async";
+import { isAbortedError, makeRequestAbortable } from "src/utils/browser";
+import { VALUE_REQUIRED } from "src/utils/constants";
+
+const initialValues: IdentityFormData = {
+  blockchain: "",
+  credentialStatusType: CredentialStatusType.Iden3OnchainSparseMerkleTreeProof2023,
```

Comment:
Since apparently we're not fixing [this](https://privadoid.slack.com/archives/C07AFTCMYGJ/p1727863701898819), let's at least default to `CredentialStatusType["Iden3commRevocationStatusV1.0"]` as the backend does.